libs/langchain-openai/src/llms.ts

```diff
@@ -14,20 +14,11 @@ import type {
   OpenAICallOptions,
   OpenAICoreRequestOptions,
   OpenAIInput,
-  LegacyOpenAIInput,
 } from "./types.js";
 import { OpenAIEndpointConfig, getEndpoint } from "./utils/azure.js";
-import { OpenAIChat, OpenAIChatCallOptions } from "./legacy.js";
 import { wrapOpenAIClientError } from "./utils/openai.js";
 
-export type {
-  AzureOpenAIInput,
-  OpenAICallOptions,
-  OpenAIInput,
-  OpenAIChatCallOptions,
-};
-
-export { OpenAIChat };
+export type { AzureOpenAIInput, OpenAICallOptions, OpenAIInput };
```

Comment:
should remove azure one if philosophy is to hide azure stuff within azure files

Reply:
Yeah good call thanks